### PR TITLE
Update to Destinations

### PR DESCRIPTION
**Pull request by artemkovalyov**
Add Auth types. A quick brush up.

@frank-essenberger , take a look if what I've added makes sense to you.

### CODE REVIEW

File: docs/js/features/connectivity/destination.mdx [hunk 1]
```diff
@@ -40,7 +40,7 @@ BusinessPartner.requestBuilder()
   .top(5)
   .execute(yourDestination);
 ```
-where `yourDestination` is a [`Destination` object](pathname:///api/1.28.1/interfaces/sap_cloud_sdk_core.destination).
+where `yourDestination` is a [`Destination` object](pathname:///api/1.32./interfaces/sap_cloud_sdk_core.destination).
```
Comment (artemkovalyov) [inline docs/js/features/connectivity/destination.mdx:43]:
```suggestion
where `yourDestination` is a [`Destination` object](pathname:///api/1.32.0/interfaces/sap_cloud_sdk_core.destination).
```

Comment (artemkovalyov) [reply docs/js/features/connectivity/destination.mdx:43]: I'll make a component to reference always the latest SDK ver.

Comment (jjtang1985) [inline docs/js/features/connectivity/destination.mdx]:
```suggestion
where `yourDestination` is a [`Destination` object](pathname:///api/1.32.0/interfaces/sap_cloud_sdk_core.destination).
```
BTW, it's so sad we don't have the latest tag

Comment (artemkovalyov) [reply docs/js/features/connectivity/destination.mdx]: I fixed it already @jjtang1985 . I'll also add a component for the lates TAG. That super easy to implement.

**Review by jjtang1985 — APPROVED** [review]

**Pull request merged**